translator/main.py

```diff
@@ -350,12 +378,14 @@ def create_dashboard_cm(
     v1 = client.CoreV1Api()
 
     dashboards_skip = translator_annotations.get("grafana_dashboards_skip_update", [])
-    target_dashboards_selector = translator_annotations.get(args_.grafana_label, None)
+    target_dashboards_selector = translator_annotations.get(
+        args_.grafana_label,
+        "grafana_dashboard=1"
```

Comment:
Should it be hard coded?

Reply:
How do you want to delete/update generated dashboards? Previous implementation did it by list that one configmap using its name. In this, we need to use labels, so you need to have default value for it if you did not set value for specified key.

Reply:
So theoretically it can delete/update the config map which is not generated by this translator, right? E.g. additional config maps added by the user with this label.

Reply:
So theoretically, I think yes.

Reply:
Why we cannot use the name as before?

Reply:
And how do you want to do it? Before, there was hard-coded name of that one specific configmap. Now, you have generated names `grafana-dashboards-generated-<filename>`.

Reply:
I understand. But why do we need the default value there when we have it in [defaults](https://github.com/dNationCloud/kubernetes-jsonnet-translator/blob/main/default_config.yaml#L19)

Reply:
If I am right, this is not default value, it is key which you are looking for.

Reply:
Ok sorry, I probably see why. But anyway, What do you think about defining it only in one place(now it is defined in two)? and also should we use some custom label for that(not an official one)?

Reply:
I agree, it would be better to define it in one place instead of two, but it still needs to be hard coded. I do not know if this one is official label or not. But if you have any suggestion for different label, it is welcome.

Reply:
See https://github.com/grafana/helm-charts/tree/main/charts/grafana#sidecar-for-dashboards.

Reply:
Or can we get all config maps but filter them out based on the generated name ? If it starts with `grafana-dashboards-generated-`?

Reply:
Or I suggest something `app.kubernetes.io/managed-by: jsonnet-translator`